bayes/vb.py

```diff
@@ -99,6 +101,29 @@ def Noninformative(cls):
         return cls(scale=1.0 / 3.0, shape=0.0)
 
 
+class BayesEncoder(json.JSONEncoder):
```

Comment:
It would be nice at some point when we start documenting the functions. Maybe a short comment on what the input is, and what the method does would help.

Reply:
Thanks for this hint, good idea! You may have a look at it again.